Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelBlueprintCreater.hpp

```diff
@@ -0,0 +1,163 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Detector/Blueprint.hpp"
+#include "Acts/Detector/KdtSurfacesProvider.hpp"
+#include "Acts/Detector/interface/IInternalStructureBuilder.hpp"
+#include "Acts/Geometry/Extent.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Geometry/VolumeBounds.hpp"
+#include "Acts/Utilities/BinningData.hpp"
+#include "Acts/Utilities/Logger.hpp"
+
+#include <map>
+#include <memory>
+#include <optional>
+#include <string>
+#include <tuple>
+#include <vector>
+
+namespace Acts {
+
+struct GeoModelTree;
+
+/// @brief This is the GeoModel blueprint creator for the Gen2 ("Acts::Experimental::Detector")
+/// geometry concept.
+///
+class GeoModelBlueprintCreater {
+ public:
+  /// The configuration struct
+  struct Config {
+    /// The detector surfaces - leave empty if filling is not done
+    /// with a kdtree sorting structure
+    std::vector<std::shared_ptr<Surface>> detectorSurfaces = {};
+    /// The binning values - see above
```

Comment:
Not sure where to look above here.

Reply:
Changed to `The binning values for the KDTree sorting`